uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/api/EventFilter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.mqtt.client.control.api;
+
+import lombok.Getter;
+import lombok.NonNull;
+
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Filter of events in event storage.
+ */
+@Getter
+public final class EventFilter {
+    private final Event.Type type;
+    private final String connectionName;
+    private final String topic;
+    private final String topicFilter;
+    private final byte[] content;
+
```

Comment:
Could you add timestampFrom and timestampTo please?

Reply:
Added in 77e7619407a9e819ebb329ac24a1e28bd4e5244d